bacon/controller/TransactionsViewController.swift

```diff
@@ -71,9 +94,63 @@ extension TransactionsViewController: UITableViewDataSource, UITableViewDelegate
         guard let cell = rawCell as? FoldingCell else {
             return rawCell
         }
-        let durations: [TimeInterval] = [0.26, 0.2, 0.2]
+        let arrayIndex = indexPath.row
+        let displayedIndex = arrayIndex + 1
+
+        // FoldingCell-specific congigurations
+        let durations: [TimeInterval] = Constants.animatoinDuration
         cell.durationsForExpandedState = durations
         cell.durationsForCollapsedState = durations
+
+        // Configure views to show data
+        let closedNumberView = cell.viewWithTag(13) as? UILabel
```

Comment:
I'm not too sure what's happening here just from reading... but should the values be refactored into the constants file?

Reply:
Oh it's configuring the subviews of the table cell to reflect the transaction information. It is very long because there are many many subviews within each cell. But should be easy to tell right? Or maybe it's the way I wrote it hmmmmmm
And yes of cos the tags can be refactored. I didn't do it because it will make each line even longer HAHA imagine cell.viewWithTag(Constans.closedNumberViewTag)